rpcs3/util/cpu_stats.cpp

```diff
@@ -126,8 +126,8 @@ namespace utils
 	{
 		total_usage = 0.0;
 
+		per_core_usage.clear();
```

Comment:
What are these random changes?
resize + fill is faster than clear + resize, according to quickbench